### ISSUE TITLE
599 cases?

### ISSUE THREAD

**Issue by Alberto-Neri** [neheller/kits23#18]
In the challenge page is written that: 
_"The dataset is composed of 599 cases with 489 allocated to the training set and 110 in the test set."_

Now, here in dataset folder there are 589 cases, with case between 300 and 399 missing, for a total of **489 cases available**. 
Does this mean that only the training set is published? Will the test set be published?

**Comment by neheller** [comment]
As a rule we do not release test set labels to anyone, but we can run metrics for individual teams on a case-by-case basis. If you'd like to get access to the test *images* for inference purposes, I'd be happy to send you a download link.

**Comment by xiaonanyang** [comment]
Hi, I was trying to do a test with this dataset, it would be so nice if I can get the test set. thank you very much for your help.

**Comment by BITEWKRER** [comment]
Hello, I am trying to do research based on KITS23 dataset, but I found that there is no image of testset, can you provide image of test dataset?

**Comment by neheller** [comment]
Apologies for the delay. Please send me an email at helle246@umn.edu so that I can send the link to you directly.

**Comment by LetiziaC14** [comment]
Hi, I came across your challenge and found it particularly relevant to my research. I have been working on solving the problem using your dataset; however, I was unable to locate a Test Set to validate my results, I sent you an email some weeks ago.
Could you please provide me with a download link or access to the Test Set?

Thanks a lot!


> As a rule we do not release test set labels to anyone, but we can run metrics for individual teams on a case-by-case basis. If you'd like to get access to the test _images_ for inference purposes, I'd be happy to send you a download link.
